networkx/algorithms/approximation/steinertree.py

```diff
@@ -1,11 +1,20 @@
 from itertools import chain
 
-from networkx.utils import pairwise, not_implemented_for
 import networkx as nx
+from networkx.utils import pairwise, not_implemented_for
```

Comment:
```suggestion
from networkx.utils import not_implemented_for, pairwise
```
To appease  the import-sort styling changes.